test/kubemark/iks/util.sh

```diff
@@ -61,34 +63,35 @@ function create-clusters {
 # Builds and pushes image to registry
 function push-image {
 	if [[ "${ISBUILD}" = "y" ]]; then
-		if ! bx cr namespaces | grep -Fq ${KUBE_NAMESPACE}; then
+		if ! bx cr namespaces | grep -Fq "${KUBE_NAMESPACE}"; then
 			echo "Creating registry namespace"
-			bx cr namespace-add ${KUBE_NAMESPACE}
-			echo "bx cr namespace-rm ${KUBE_NAMESPACE}" >> ${RESOURCE_DIRECTORY}/iks-namespacelist.sh
+			bx cr namespace-add "${KUBE_NAMESPACE}"
+			echo "bx cr namespace-rm ${KUBE_NAMESPACE}" >> "${RESOURCE_DIRECTORY}/iks-namespacelist.sh"
 		fi
-		docker build -t ${KUBEMARK_INIT_TAG} ${KUBEMARK_IMAGE_LOCATION}
-		docker tag ${KUBEMARK_INIT_TAG} ${KUBEMARK_IMAGE_REGISTRY}${KUBE_NAMESPACE}/${PROJECT}:${KUBEMARK_IMAGE_TAG}
-		docker push ${KUBEMARK_IMAGE_REGISTRY}${KUBE_NAMESPACE}/${PROJECT}:${KUBEMARK_IMAGE_TAG}
+		docker build -t "${KUBEMARK_INIT_TAG}" "${KUBEMARK_IMAGE_LOCATION}"
+		docker tag "${KUBEMARK_INIT_TAG}" "${KUBEMARK_IMAGE_REGISTRY}${KUBE_NAMESPACE}/${PROJECT}:${KUBEMARK_IMAGE_TAG}"
+		docker push "${KUBEMARK_IMAGE_REGISTRY}${KUBE_NAMESPACE}/${PROJECT}:${KUBEMARK_IMAGE_TAG}"
 		echo "Image pushed"
 	else
-		KUBEMARK_IMAGE_REGISTRY=$(echo "brandondr96")
+		KUBEMARK_IMAGE_REGISTRY="brandondr96"
 		KUBE_NAMESPACE=""
 	fi
 }
 
 # Allow user to use existing clusters if desired
 function choose-clusters {
 	echo -n -e "Do you want to use custom clusters? [y/N]${color_cyan}>${color_norm} "
-	read USE_EXISTING
+	read -r USE_EXISTING
```

Comment:
For reference [SC2162](https://github.com/koalaman/shellcheck/wiki/SC2162) - _"read without -r will mangle backslashes"_